charts/tidb-backup/templates/scripts/_start_restore.sh.tpl

```diff
@@ -9,7 +9,7 @@ downloader \
   --cloud=gcp \
   --bucket={{ .Values.gcp.bucket }} \
   --srcDir=${BACKUP_NAME} \
-  --destDir=${dirname}
+  --destDir=/data
```

Comment:
use `/data` instead of `${dirname}`?

Reply:
if destDir is ${dirname}, loader 
```
/loader \
  -d=${dirname}
```  

will be

```
-d=${dirname}/${BACKUP_NAME}
```